lib/jsdom/living/nodes/Node-impl.js

```diff
@@ -128,6 +128,15 @@ class NodeImpl extends EventTargetImpl {
     this._memoizedQueries = {};
     this._registeredObserverList = [];
     this._referencedRanges = new Set();
+    this._inert = false;
+  }
+
+  get inert() {
```

Comment:
These getters and setters are not necessary; just remove the underscore from the declaration.